test/Agreement.js

```diff
@@ -66,9 +66,13 @@ contract(
       expect(event.args.accepter).to.eq(signer);
     }
 
-    async function expectAgreementAccepted(signer) {
-      const signed = await agreement.isAgreementSignedBy.call(signer);
-      expect(signed).to.be.true;
+    async function expectAgreementAccepted(signer, firstTx) {
```

Comment:
why is this parameter named `firstTx`? I think name like `signTx` makes more sense